src/main/ApplicationUtils.cpp

```diff
@@ -40,7 +40,8 @@ runWithConfig(Config cfg)
     }
 
     LOG(INFO) << "Starting stellar-core " << STELLAR_CORE_VERSION;
-    VirtualClock clock(VirtualClock::REAL_TIME);
+    VirtualClock clock(cfg.MANUAL_CLOSE ? VirtualClock::VIRTUAL_TIME
```

Comment:
This doesn't seem to implement everything in #1760.

Reply:
Ah I see, it's in the next commit. Probably should squash them.

Reply:
Squashed.